routes/userland/show.js

```diff
@@ -1,33 +1,40 @@
-module.exports = (req, res) => {
+module.exports = (req, res, next) => {
   const currentReport = req.params[0]
-  const report = require(`../../data/userland/${currentReport}`)
-  let blacklist
+  try {
+    const report = require(`../../data/userland/${currentReport}`)
+    let blacklist
 
-  report.isPackage = report.collectionType === 'Package'
-  report.isRepository = report.collectionType === 'Repository'
-  report.isnpmUser = report.collectionType === 'npmUser'
-  report.isGitHubUser = report.collectionType === 'GithubUser'
+    report.isPackage = report.collectionType === 'Package'
+    report.isRepository = report.collectionType === 'Repository'
+    report.isnpmUser = report.collectionType === 'npmUser'
+    report.isGitHubUser = report.collectionType === 'GithubUser'
 
-  if (report.isGitHubUser) {
-    blacklist = [
-      'greenkeeperio-bot',
-      'ember-tomster',
-      'gitter-badger',
-      'invalid-email-address',
-      'waffle-iron'
-    ]
-  } else if (report.isnpmUser) {
-    blacklist = [
-      'uupaa',
-      'etc-etc-etc'
-    ]
-  }
+    if (report.isGitHubUser) {
+      blacklist = [
+        'greenkeeperio-bot',
+        'ember-tomster',
+        'gitter-badger',
+        'invalid-email-address',
+        'waffle-iron'
+      ]
+    } else if (report.isnpmUser) {
+      blacklist = [
+        'uupaa',
+        'etc-etc-etc'
+      ]
+    }
 
-  const context = Object.assign(req.context, {
-    report: report,
-    items: report.collection,
-    blacklist: blacklist
-  })
+    const context = Object.assign(req.context, {
+      report: report,
+      items: report.collection,
+      blacklist: blacklist,
+      page: {
+        title: `${report.title} | Electron`
+      }
+    })
 
-  res.render('userland/show', context)
+    res.render('userland/show', context)
+  } catch (e) {
```

Comment:
Rather than catching this entire block of code, can we just detect a 404 earlier and then `return next()`?